collector.html

```diff
@@ -16,6 +16,18 @@
 -->
 
 <script type="text/x-red" data-template-name="collector">
+
+    <style>
+        .checkbox {
+            display: inline-block !important;
+            width: auto !important;
+            vertical-align: top !important;
+        }
+        .checkbox-label {
+            width: 70% !important;
+        }
+    </style>
```

Comment:
I don't think CSS should be needed for a standard Node-RED form.

Was this to make the form look neater?

Reply:
> I don't think CSS should be needed for a standard Node-RED form.

Ah yes my mistake - it shouldn't be there.

> Was this to make the form look neater?

Initially yes.

### TL;DR
Would you mind changing the PR to simply remove this CSS - it was introduced when I first attempted checkboxes in the UI and now isn't necessary?

### Slight longer explanation
I initially tried using checkboxes in the UI but I then realised there needed to be an explanation for when merging wasn't enabled and with a checkbox, there wasn't a 'term' to describe this (merging is on or off).  So I instead switched to a menu as per your other suggestion.  And I forgot to remove the css. 

Cheers,
D.